site/features.md

```diff
@@ -108,6 +109,19 @@ IP addresses of external services the hosts or containers need to
 connect to. The same IP range must be used everywhere, and the
 individual IP addresses must, of course, be unique.
 
+### <a name="naming-and-discovery"></a>Naming and discovery
+
+Application containers automatically register themselves for discovery
+simply by giving them a container name when they are launched:
```

Comment:
"Named containers are automatically registered in weaveDNS(link), which makes them discoverable through simple name lookups:"